src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -6,6 +6,9 @@ namespace NServiceBus.Hosting.Helpers
     using System.Linq;
     using System.Reflection;
     using System.Runtime.CompilerServices;
+#if NETCOREAPP2_1
```

Comment:
I'm thinking a more general `NETCOREAPP` makes more sense than `NETCOREAPP2_1` for the ifdefs.